bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
Just checked the last 4 commits and I don't see where you applied something. Also don't see any open PRs except this one. Can you clarify?

----

One thing we could do to alleviate the confusion between xmlrpc and buglist would be to change line 361 to use the correct script:

    - basequery = bzserver + "bugs/buglist.cgi?status_whiteboard=RHT"
    + basequery = bzserver + "bugs/xmlrpc.cgi?status_whiteboard=RHT"

Then on lines 388-390 you wouldn't be passing your parameters to a DIFFERENT script than is echoed in the DEBUG statement:

    388: bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
    389:
    390: print "[DEBUG] " + "Querying bugzilla: " + query